docs/zkapps/how-to-write-a-zkapp.mdx

```diff
@@ -35,54 +35,71 @@ npm install -g zkapp-cli
-2. **Run tests:** Run `npm run test`. Tests are written using <a
-   href="https://jestjs.io/">Jest</a>. After running this command, you
-   see all tests pass. You can also run `npm run testw` to run tests in watch
-   mode, so tests are automatically re-run tests when you save changes to your code.
-3. **Build the example:** Run `npm run build` to compile your TypeScript
-   into JavaScript inside the project’s `/build` directory.
-4. **Deploy to Testnet:** Run `zk config`. The command walks you through adding a
-   network alias to your project’s `config.json`. For Berkeley Testnet, use `berkeley` as the name, `0.1` for the fee, and
-   `https://proxy.berkeley.minaexplorer.com/graphql` for the url. Then run `zk deploy` and follow the prompts. See the [how to deploy a zkApp](how-to-deploy-a-zkapp) page for further details.
+5. Deploy to Testnet: `zk deploy`
 
-You can view a list of <a href="https://github.com/o1-labs/zkapp-cli/tree/main/examples/">all available examples here</a>.
+  Follow the prompts. For details, see [how to deploy a zkApp](how-to-deploy-a-zkapp).
 
 #### Option B: Start your own project
 
-1. **Install:** Run `zk project <myproj>`. Type `ls` and hit enter to see the
-   newly created project structure. _Note_: If you want to directly scaffold an UI with your new zkApp, you can do so by adding the `--ui` flag to the previous command, like `zk project <myproj> --ui=<framework>`. You can currently choose from the following frameworks: `svelte`, `next` or `nuxt` 
-2. **Run tests:** Run `npm run test`. Tests are written using <a
-   href="https://jestjs.io/">Jest</a>. After running this command, you can
-   see that all tests pass. You can also run `npm run testw` to run tests in watch
-   mode, so tests are automatically re-run you save changes to your code.
-3. **Build:** Run `npm run build` to compile your TypeScript code into
-   JavaScript inside the project’s `/build`.
-4. **Deploy to Testnet:** Run `zk config`. The command walks you through adding a
-   network alias to your project’s `config.json`. For Berkeley Testnet, use `berkeley` as the name, `0.1` for the fee, and
-   `https://proxy.berkeley.minaexplorer.com/graphql` for the url. Then run `zk deploy` and follow the prompts. See the [how to deploy a zkApp](how-to-deploy-a-zkapp) page for further details.
-5. **Deploy to Mainnet:** (Coming soon.)
+Using the steps above, create your own project using `zk project <myproj>`.
 
 ### Writing your smart contract
 
-The goal of this section is to explain the concepts that you need to understand how to write a zero-knowledge-based smart contract.
+This section explain the concepts to understand how to write a zero knowledge-based smart contract.
 
-If you haven’t yet read the [how zkApps work](how-zkapps-work) pages, please read it now so that this section makes sense.
+If you haven't yet read [how zkApps work](how-zkapps-work) pages, please read it now to build your foundational knowledge.
 
 ##### SnarkyJS
```

Comment:
this content will be restructed, so defer grooming and substantive edits from here down (lots of work!)